app/models/health/enrollment_disenrollment.rb

```diff
@@ -6,9 +6,10 @@
 
 module Health
   class EnrollmentDisenrollment
-    def initialize(start_date, end_date, aco_ids, enrollment_reasons)
+    def initialize(start_date, end_date, effective_date, aco_ids, enrollment_reasons)
```

Comment:
Not sure in how many places we use this, but maybe these should be named parameters?